client/rest-high-level/build.gradle

```diff
@@ -64,3 +82,32 @@ forbiddenApisMain {
   signaturesURLs += [PrecommitTasks.getResource('/forbidden/http-signatures.txt')]
   signaturesURLs += [file('src/main/resources/forbidden/rest-high-level-signatures.txt').toURI().toURL()]
 }
+
+javadoc {
+  // Bundle javadoc from all bundled projects into this project's javadoc
+  configurations.bundled.dependencies.all { Dependency dep ->
+    Project p = dependencyToProject(dep)
+    if (p != null) {
+      evaluationDependsOn(p.path)
```

Comment:
I couldn't find a way to make this sufficiently lazy to avoid forcing evaluation of the other project. I believe I could use `afterProjectsEvaluated` but this feels much more precise.

Reply:
Shouldn't this be something in BuildPlugin? Seems like we would want this for any cross project dependencies?

Reply:
We only want this if we want to combine that javadocs from a bunch of different projects into one place. It feels like something we might only want in the high level rest client. Maybe into the plugin API when we make a jar for it. If we make a jar for it.

Reply:
So after thinking about this some more, maybe! I think we might want this if we detect that the shadow plugin is around.